test/src/e2e.dynval/setup.ts

```diff
@@ -464,3 +464,36 @@ export function setTermTestTimeout(
         }
     };
 }
+
+export async function termThatIncludeTransaction(
+    sdk: SDK,
+    txHash: H256
+): Promise<number> {
+    // Diable lint rule to extract let and const variable from Promise.all.
+    // tslint:disable-next-line:prefer-const
+    let [termMetadata, transaction] = await Promise.all([
+        stake.getTermMetadata(sdk),
```

Comment:
You can utilize `blockNumber` parameter

Reply:
Good!
I changed the code to use the blockNumber parameter.